tests/functional/spec/features/dynamic_answers_list_value_source.spec.js

```diff
@@ -162,6 +162,24 @@ describe("Dynamic answers list value source", () => {
     await expect(await $$(OnlineShoppingPage.labels())[1].getText()).toBe("Percentage of online shopping at Aldi");
     await expect(await $$(OnlineShoppingPage.labels()).length).toBe(4);
   });
+
+  it("Given the minimum value is set to 1000.99, When the maximum amount of spending entered should be between 1000.99 and 10000 and 1000.98 is entered, Then the value should not be accepted", async () => {
```

Comment:
To build on the above, we should update `numbers.spec.js`. The test should be relevant to the schema it is in.